locale/en-US/tbSync.strings

```diff
@@ -17,22 +17,22 @@ prompt.UnlockSettings=Changing any of the predefined server settings could lead
 
 account=TbSync account [##accountname##]
 
-accountacctions.delete=Delete account "##accountname##"
-accountacctions.sync=Synchronize account "##accountname##"
-accountacctions.enable=Enable account "##accountname##" & try to connect to server
-accountacctions.disable=Disable account "##accountname##"
+accountacctions.delete=Delete account “##accountname##”
+accountacctions.sync=Synchronize account “##accountname##”
+accountacctions.enable=Enable account “##accountname##” and try to connect to server
```

Comment:
I want to keep the &

Reply:
bad english using & instead of and

Reply:
It's okay in informal writing. Really depends how formal one imagines this to be!

Reply:
Sorry, we thought this was planned as professional software for the enterprise.